File: tests/eval-fixtures/cfg-core/repo/go/cfg.go
package cfgcore

func helper(value int) int {
	return value + 1
}

func Route(input int) int {
	total := input
	if input > 0 && helper(input) > 1 {
		total = helper(total)
	}
	for total < 10 {
		total = total + 1
	}
	defer helper(total)
	if total > 50 {
		panic(total)
		return 99
	}
	return total
}

File: tests/eval-fixtures/cfg-core/repo/ts/cfg.ts
function helper(value: number): number {
  return value + 1;
}

export function route(input: number, options?: { enabled?: boolean }): number {
  let total = input;
  if (options?.enabled && (input ?? 0) > 0) {
    total = helper(total);
  }
  for (; total < 10; total++) {
    total = helper(total);
  }
  try {
    if (total > 50) {
      throw new Error("too high");
      return -1;
    }
  } finally {
    total = helper(total);
  }
  return total;
}
